entry/src/ohosTest/java/me/jfenn/hmosutils/AnimatedColorTest.java

```diff
@@ -0,0 +1,43 @@
+package me.jfenn.hmosutils;
```

Comment:
Applib group copyright can be added for all the test files.

Reply:
Done